### PR TITLE
feat: Tabbing Order E2E Tests - Product Page

### PR DESCRIPTION
**Pull request by Zeyber**
Closes: #4200

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts [hunk 1]
```diff
@@ -84,6 +84,18 @@ export function checkElement(tabElement: TabElement) {
         .should('have.attr', 'formcontrolname', tabElement.value);
       break;
     }
+    case TabbingOrderTypes.H3: {
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: is h3 an interactable element?

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts [hunk 2]
```diff
@@ -84,6 +84,18 @@ export function checkElement(tabElement: TabElement) {
...
+      cy.focused()
+        .get('h3')
+        .should('contain', tabElement.value);
+      return;
+    }
+    case TabbingOrderTypes.CX_MEDIA: {
```
Comment (kacperknapik) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: can we check something different than images' src? every change to the backend will brake it. too fragile imho

Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: Like what?

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts (hunk 1 above)
Comment (Zeyber) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: In this situation, yes.

File: projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts (hunk 2 above)
Comment (kacperknapik) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.ts]: I just noticed that the `` elements have `alt` attributes, which contain name of the product, so we can go this way. much safer tbh

**Pull request merged**